docs/Installing.asciidoc

```diff
@@ -128,7 +128,7 @@ package.
 
 The easiest way to install openQA is from distribution packages.
 
-- For openSUSE, packages are available for Leap and Tumbleweed
+- For SUSE Linux Enterprise (SLE)/openSUSE, packages are available for Leap and Tumbleweed and SLE
```

Comment:
I see some modules later in the text, so we are talking about SLE 15 and newer, right? But we do not list Leap versions either, so I guess this is ok.

Reply:
yes, I want to avoid mentioning specific versions. There can always be version specific differences.

Reply:
I added one sentence

```
It is suggested to always use the latest release version of operating systems.
```

That's the best I could find trying to put a statement into the text. I would simply not state anything regarding "support installation".